klippy/extras/bme280.py

```diff
@@ -319,15 +322,17 @@ def read_calibration_data_bmp180(calib_data_1):
 
             self.sample_timer = self.reactor.register_timer(self._sample_bmp388)
         else:
-            self.max_sample_time = \
-                (1.25 + (2.3 * self.os_temp) + ((2.3 * self.os_pres) + .575)
-                 + ((2.3 * self.os_hum) + .575)) / 1000
+            if self.chip_type == 'BME280':
```

Comment:
Minor nitpick, but I believe this conditional block can be an extension of its parent block.